src/js/utils/browser.js

```diff
@@ -25,12 +25,15 @@ define([
     };
 
     browser.isFF = _browserCheck(/firefox/i);
-    browser.isChrome = _browserCheck(/chrome/i);
     browser.isIPod = _browserCheck(/iP(hone|od)/i);
     browser.isIPad = _browserCheck(/iPad/i);
     browser.isSafari602 = _browserCheck(/Macintosh.*Mac OS X 10_8.*6\.0\.\d* Safari/i);
+    browser.isEdge = _browserCheck(/edge/i);
```

Comment:
That's it? Seems a little... short and likely to match parts of other words or strings in other user-agents.